[2025-01-14T18:38:58.099+0000] {processor.py:186} INFO - Started process (PID=252) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:38:58.101+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:38:58.105+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:38:58.104+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:39:00.505+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:39:00.498+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_v1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_v1.py", line 4, in <module>
    from etl import transform, check_and_extract_data, load_data
ImportError: cannot import name 'transform' from 'etl' (/opt/airflow/dags/etl.py)
[2025-01-14T18:39:00.509+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:39:00.531+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 2.440 seconds
[2025-01-14T18:39:30.897+0000] {processor.py:186} INFO - Started process (PID=260) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:39:30.899+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:39:30.902+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:39:30.902+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:39:31.698+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:39:31.692+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_v1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_v1.py", line 4, in <module>
    from etl import transform, check_and_extract_data, load_data
ImportError: cannot import name 'transform' from 'etl' (/opt/airflow/dags/etl.py)
[2025-01-14T18:39:31.701+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:39:31.724+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.836 seconds
[2025-01-14T18:40:01.953+0000] {processor.py:186} INFO - Started process (PID=268) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:40:01.956+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:40:01.960+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:40:01.960+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:40:02.782+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:40:02.777+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_v1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_v1.py", line 4, in <module>
    from etl import transform, check_and_extract_data, load_data
ImportError: cannot import name 'transform' from 'etl' (/opt/airflow/dags/etl.py)
[2025-01-14T18:40:02.784+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:40:02.807+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.863 seconds
[2025-01-14T18:40:32.871+0000] {processor.py:186} INFO - Started process (PID=276) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:40:32.873+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:40:32.876+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:40:32.876+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:40:33.813+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:40:33.805+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_v1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_v1.py", line 4, in <module>
    from etl import transform, check_and_extract_data, load_data
ImportError: cannot import name 'transform' from 'etl' (/opt/airflow/dags/etl.py)
[2025-01-14T18:40:33.815+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:40:33.848+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.986 seconds
[2025-01-14T18:41:04.157+0000] {processor.py:186} INFO - Started process (PID=284) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:41:04.158+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:41:04.162+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:41:04.161+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:41:04.997+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:41:04.991+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_v1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_v1.py", line 4, in <module>
    from etl import transform, check_and_extract_data, load_data
ImportError: cannot import name 'transform' from 'etl' (/opt/airflow/dags/etl.py)
[2025-01-14T18:41:05.000+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:41:05.026+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.883 seconds
[2025-01-14T18:41:35.320+0000] {processor.py:186} INFO - Started process (PID=292) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:41:35.322+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:41:35.328+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:41:35.328+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:41:36.658+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:41:36.624+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_v1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_v1.py", line 4, in <module>
    from etl import transform, check_and_extract_data, load_data
ImportError: cannot import name 'transform' from 'etl' (/opt/airflow/dags/etl.py)
[2025-01-14T18:41:36.668+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:41:36.713+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 1.407 seconds
[2025-01-14T18:42:06.945+0000] {processor.py:186} INFO - Started process (PID=300) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:42:06.947+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:42:06.952+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:42:06.952+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:42:07.679+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:42:07.672+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_v1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_v1.py", line 4, in <module>
    from etl import transform, check_and_extract_data, load_data
ImportError: cannot import name 'transform' from 'etl' (/opt/airflow/dags/etl.py)
[2025-01-14T18:42:07.682+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:42:07.702+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.773 seconds
[2025-01-14T18:42:37.866+0000] {processor.py:186} INFO - Started process (PID=308) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:42:37.868+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:42:37.873+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:42:37.872+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:42:38.741+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:42:38.734+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_v1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_v1.py", line 4, in <module>
    from etl import transform, check_and_extract_data, load_data
ImportError: cannot import name 'transform' from 'etl' (/opt/airflow/dags/etl.py)
[2025-01-14T18:42:38.746+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:42:38.776+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.918 seconds
[2025-01-14T18:43:09.016+0000] {processor.py:186} INFO - Started process (PID=322) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:43:09.017+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:43:09.020+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:43:09.020+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:43:09.769+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:43:09.761+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_v1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_v1.py", line 4, in <module>
    from etl import transform, check_and_extract_data, load_data
ImportError: cannot import name 'transform' from 'etl' (/opt/airflow/dags/etl.py)
[2025-01-14T18:43:09.773+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:43:09.797+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.791 seconds
[2025-01-14T18:43:40.072+0000] {processor.py:186} INFO - Started process (PID=330) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:43:40.074+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:43:40.078+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:43:40.078+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:43:40.930+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:43:40.922+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_v1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_v1.py", line 4, in <module>
    from etl import transform, check_and_extract_data, load_data
ImportError: cannot import name 'transform' from 'etl' (/opt/airflow/dags/etl.py)
[2025-01-14T18:43:40.934+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:43:40.961+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.902 seconds
[2025-01-14T18:44:11.325+0000] {processor.py:186} INFO - Started process (PID=338) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:44:11.326+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:44:11.330+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:44:11.329+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:44:12.078+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:44:12.071+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_v1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_v1.py", line 4, in <module>
    from etl import transform, check_and_extract_data, load_data
ImportError: cannot import name 'transform' from 'etl' (/opt/airflow/dags/etl.py)
[2025-01-14T18:44:12.081+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:44:12.109+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.792 seconds
[2025-01-14T18:44:42.388+0000] {processor.py:186} INFO - Started process (PID=346) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:44:42.390+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:44:42.397+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:44:42.396+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:44:43.254+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:44:43.246+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_v1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_v1.py", line 4, in <module>
    from etl import transform, check_and_extract_data, load_data
ImportError: cannot import name 'transform' from 'etl' (/opt/airflow/dags/etl.py)
[2025-01-14T18:44:43.259+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:44:43.294+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.922 seconds
[2025-01-14T18:45:13.592+0000] {processor.py:186} INFO - Started process (PID=354) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:45:13.593+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:45:13.597+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:45:13.596+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:45:14.266+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:45:14.260+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_v1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_v1.py", line 4, in <module>
    from etl import transform, check_and_extract_data, load_data
ImportError: cannot import name 'transform' from 'etl' (/opt/airflow/dags/etl.py)
[2025-01-14T18:45:14.269+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:45:14.295+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.712 seconds
[2025-01-14T18:45:44.563+0000] {processor.py:186} INFO - Started process (PID=362) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:45:44.565+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:45:44.568+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:45:44.567+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:45:45.351+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:45:45.343+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_v1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_v1.py", line 4, in <module>
    from etl import transform, check_and_extract_data, load_data
ImportError: cannot import name 'transform' from 'etl' (/opt/airflow/dags/etl.py)
[2025-01-14T18:45:45.354+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:45:45.378+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.824 seconds
[2025-01-14T18:46:15.699+0000] {processor.py:186} INFO - Started process (PID=370) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:46:15.700+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:46:15.703+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:46:15.702+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:46:16.387+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:46:16.380+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_v1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_v1.py", line 4, in <module>
    from etl import transform, check_and_extract_data, load_data
ImportError: cannot import name 'transform' from 'etl' (/opt/airflow/dags/etl.py)
[2025-01-14T18:46:16.390+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:46:16.417+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.727 seconds
[2025-01-14T18:46:46.870+0000] {processor.py:186} INFO - Started process (PID=378) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:46:46.873+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:46:46.886+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:46:46.885+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:46:47.941+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:46:47.936+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_v1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_v1.py", line 4, in <module>
    from etl import transform, check_and_extract_data, load_data
ImportError: cannot import name 'transform' from 'etl' (/opt/airflow/dags/etl.py)
[2025-01-14T18:46:47.943+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:46:47.967+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 1.177 seconds
[2025-01-14T18:47:18.203+0000] {processor.py:186} INFO - Started process (PID=386) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:47:18.205+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:47:18.208+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:47:18.208+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:47:19.047+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:47:19.041+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_v1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_v1.py", line 4, in <module>
    from etl import transform, check_and_extract_data, load_data
ImportError: cannot import name 'transform' from 'etl' (/opt/airflow/dags/etl.py)
[2025-01-14T18:47:19.049+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:47:19.072+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.877 seconds
[2025-01-14T18:47:49.347+0000] {processor.py:186} INFO - Started process (PID=394) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:47:49.349+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:47:49.352+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:47:49.352+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:47:50.097+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:47:50.092+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_v1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_v1.py", line 4, in <module>
    from etl import transform, check_and_extract_data, load_data
ImportError: cannot import name 'transform' from 'etl' (/opt/airflow/dags/etl.py)
[2025-01-14T18:47:50.100+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:47:50.120+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.782 seconds
[2025-01-14T18:48:20.473+0000] {processor.py:186} INFO - Started process (PID=402) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:48:20.474+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:48:20.478+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:48:20.478+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:48:21.251+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:48:21.244+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_v1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_v1.py", line 4, in <module>
    from etl import transform, check_and_extract_data, load_data
ImportError: cannot import name 'transform' from 'etl' (/opt/airflow/dags/etl.py)
[2025-01-14T18:48:21.254+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:48:21.284+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.819 seconds
[2025-01-14T18:48:51.562+0000] {processor.py:186} INFO - Started process (PID=410) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:48:51.564+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:48:51.568+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:48:51.568+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:48:52.253+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:48:52.245+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_v1.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_v1.py", line 4, in <module>
    from etl import transform, check_and_extract_data, load_data
ImportError: cannot import name 'transform' from 'etl' (/opt/airflow/dags/etl.py)
[2025-01-14T18:48:52.256+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:48:52.278+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.724 seconds
[2025-01-14T18:49:03.559+0000] {processor.py:186} INFO - Started process (PID=412) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:49:03.561+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:49:03.567+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:49:03.566+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:49:04.442+0000] {processor.py:925} INFO - DAG(s) 'airflow_dag_amazon' retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:49:04.780+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:49:04.779+0000] {override.py:1911} INFO - Created Permission View: can read on DAG:airflow_dag_amazon
[2025-01-14T18:49:04.797+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:49:04.796+0000] {override.py:1911} INFO - Created Permission View: can delete on DAG:airflow_dag_amazon
[2025-01-14T18:49:04.812+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:49:04.810+0000] {override.py:1911} INFO - Created Permission View: can edit on DAG:airflow_dag_amazon
[2025-01-14T18:49:04.830+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:49:04.829+0000] {override.py:1911} INFO - Created Permission View: can read on DAG Run:airflow_dag_amazon
[2025-01-14T18:49:04.845+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:49:04.845+0000] {override.py:1911} INFO - Created Permission View: menu access on DAG Run:airflow_dag_amazon
[2025-01-14T18:49:04.864+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:49:04.863+0000] {override.py:1911} INFO - Created Permission View: can delete on DAG Run:airflow_dag_amazon
[2025-01-14T18:49:04.879+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:49:04.878+0000] {override.py:1911} INFO - Created Permission View: can create on DAG Run:airflow_dag_amazon
[2025-01-14T18:49:04.881+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:49:04.880+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-14T18:49:04.910+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:49:04.908+0000] {dag.py:3262} INFO - Creating ORM DAG for airflow_dag_amazon
[2025-01-14T18:49:04.912+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:49:04.911+0000] {dag.py:4180} INFO - Setting next_dagrun for airflow_dag_amazon to None, run_after=None
[2025-01-14T18:49:04.951+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 1.402 seconds
[2025-01-14T18:49:35.225+0000] {processor.py:186} INFO - Started process (PID=420) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:49:35.226+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:49:35.229+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:49:35.229+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:49:36.197+0000] {processor.py:925} INFO - DAG(s) 'airflow_dag_amazon' retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:49:36.228+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:49:36.228+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-14T18:49:36.246+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:49:36.245+0000] {dag.py:4180} INFO - Setting next_dagrun for airflow_dag_amazon to None, run_after=None
[2025-01-14T18:49:36.269+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 1.053 seconds
[2025-01-14T18:50:06.541+0000] {processor.py:186} INFO - Started process (PID=428) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:50:06.543+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:50:06.546+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:50:06.545+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:50:07.724+0000] {processor.py:925} INFO - DAG(s) 'airflow_dag_amazon' retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:50:07.763+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:50:07.762+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-14T18:50:07.810+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:50:07.809+0000] {dag.py:4180} INFO - Setting next_dagrun for airflow_dag_amazon to None, run_after=None
[2025-01-14T18:50:07.852+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 1.321 seconds
[2025-01-14T18:50:38.155+0000] {processor.py:186} INFO - Started process (PID=436) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:50:38.157+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:50:38.164+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:50:38.163+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:50:39.510+0000] {processor.py:925} INFO - DAG(s) 'airflow_dag_amazon' retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:50:39.549+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:50:39.548+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-14T18:50:39.568+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:50:39.568+0000] {dag.py:4180} INFO - Setting next_dagrun for airflow_dag_amazon to None, run_after=None
[2025-01-14T18:50:39.615+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 1.471 seconds
[2025-01-14T18:51:09.983+0000] {processor.py:186} INFO - Started process (PID=450) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:51:09.986+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:51:09.989+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:51:09.989+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:51:10.840+0000] {processor.py:925} INFO - DAG(s) 'airflow_dag_amazon' retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:51:10.867+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:51:10.866+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-14T18:51:10.882+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:51:10.882+0000] {dag.py:4180} INFO - Setting next_dagrun for airflow_dag_amazon to None, run_after=None
[2025-01-14T18:51:10.905+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.932 seconds
[2025-01-14T18:51:41.249+0000] {processor.py:186} INFO - Started process (PID=458) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:51:41.250+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:51:41.258+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:51:41.256+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:51:42.308+0000] {processor.py:925} INFO - DAG(s) 'airflow_dag_amazon' retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:51:42.345+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:51:42.344+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-14T18:51:42.366+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:51:42.366+0000] {dag.py:4180} INFO - Setting next_dagrun for airflow_dag_amazon to None, run_after=None
[2025-01-14T18:51:42.395+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 1.155 seconds
[2025-01-14T18:52:13.168+0000] {processor.py:186} INFO - Started process (PID=466) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:52:13.169+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:52:13.173+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:52:13.173+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:52:14.085+0000] {processor.py:925} INFO - DAG(s) 'airflow_dag_amazon' retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:52:14.119+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:52:14.119+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-14T18:52:14.138+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:52:14.138+0000] {dag.py:4180} INFO - Setting next_dagrun for airflow_dag_amazon to None, run_after=None
[2025-01-14T18:52:14.169+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 1.011 seconds
[2025-01-14T18:52:44.720+0000] {processor.py:186} INFO - Started process (PID=474) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:52:44.722+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:52:44.726+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:52:44.725+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:52:45.670+0000] {processor.py:925} INFO - DAG(s) 'airflow_dag_amazon' retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:52:45.701+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:52:45.700+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-14T18:52:45.724+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:52:45.723+0000] {dag.py:4180} INFO - Setting next_dagrun for airflow_dag_amazon to None, run_after=None
[2025-01-14T18:52:45.748+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 1.037 seconds
[2025-01-14T18:53:15.897+0000] {processor.py:186} INFO - Started process (PID=482) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:53:15.899+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:53:15.902+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:53:15.902+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:53:17.541+0000] {processor.py:925} INFO - DAG(s) 'airflow_dag_amazon' retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:53:17.581+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:53:17.579+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-14T18:53:17.601+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:53:17.601+0000] {dag.py:4180} INFO - Setting next_dagrun for airflow_dag_amazon to None, run_after=None
[2025-01-14T18:53:17.633+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 1.745 seconds
[2025-01-14T18:53:47.816+0000] {processor.py:186} INFO - Started process (PID=490) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:53:47.819+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:53:47.825+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:53:47.824+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:53:48.868+0000] {processor.py:925} INFO - DAG(s) 'airflow_dag_amazon' retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:53:48.907+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:53:48.906+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-14T18:53:48.929+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:53:48.929+0000] {dag.py:4180} INFO - Setting next_dagrun for airflow_dag_amazon to None, run_after=None
[2025-01-14T18:53:48.960+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 1.162 seconds
[2025-01-14T18:54:19.175+0000] {processor.py:186} INFO - Started process (PID=498) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:54:19.177+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:54:19.183+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:54:19.183+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:54:19.996+0000] {processor.py:925} INFO - DAG(s) 'airflow_dag_amazon' retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:54:20.047+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:54:20.044+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-14T18:54:20.083+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:54:20.082+0000] {dag.py:4180} INFO - Setting next_dagrun for airflow_dag_amazon to None, run_after=None
[2025-01-14T18:54:20.133+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.976 seconds
[2025-01-14T18:54:50.441+0000] {processor.py:186} INFO - Started process (PID=506) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:54:50.443+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:54:50.448+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:54:50.448+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:54:51.197+0000] {processor.py:925} INFO - DAG(s) 'airflow_dag_amazon' retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:54:51.233+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:54:51.232+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-14T18:54:51.255+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:54:51.255+0000] {dag.py:4180} INFO - Setting next_dagrun for airflow_dag_amazon to None, run_after=None
[2025-01-14T18:54:51.288+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.861 seconds
[2025-01-14T18:55:21.639+0000] {processor.py:186} INFO - Started process (PID=514) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:55:21.641+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:55:21.644+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:55:21.644+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:55:22.551+0000] {processor.py:925} INFO - DAG(s) 'airflow_dag_amazon' retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:55:22.587+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:55:22.586+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-14T18:55:22.608+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:55:22.608+0000] {dag.py:4180} INFO - Setting next_dagrun for airflow_dag_amazon to None, run_after=None
[2025-01-14T18:55:22.632+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 1.030 seconds
[2025-01-14T18:55:52.934+0000] {processor.py:186} INFO - Started process (PID=522) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:55:52.936+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:55:52.942+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:55:52.941+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:55:53.594+0000] {processor.py:925} INFO - DAG(s) 'airflow_dag_amazon' retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:55:53.634+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:55:53.631+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-14T18:55:53.654+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:55:53.653+0000] {dag.py:4180} INFO - Setting next_dagrun for airflow_dag_amazon to None, run_after=None
[2025-01-14T18:55:53.683+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.759 seconds
[2025-01-14T18:56:24.001+0000] {processor.py:186} INFO - Started process (PID=530) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:56:24.003+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:56:24.008+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:56:24.008+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:56:24.619+0000] {processor.py:925} INFO - DAG(s) 'airflow_dag_amazon' retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:56:24.651+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:56:24.651+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-14T18:56:24.670+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:56:24.670+0000] {dag.py:4180} INFO - Setting next_dagrun for airflow_dag_amazon to None, run_after=None
[2025-01-14T18:56:24.697+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.716 seconds
[2025-01-14T18:56:55.090+0000] {processor.py:186} INFO - Started process (PID=538) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:56:55.091+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:56:55.097+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:56:55.097+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:56:56.111+0000] {processor.py:925} INFO - DAG(s) 'airflow_dag_amazon' retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:56:56.192+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:56:56.191+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-14T18:56:56.227+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:56:56.226+0000] {dag.py:4180} INFO - Setting next_dagrun for airflow_dag_amazon to None, run_after=None
[2025-01-14T18:56:56.257+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 1.185 seconds
[2025-01-14T18:57:26.317+0000] {processor.py:186} INFO - Started process (PID=546) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:57:26.319+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:57:26.323+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:57:26.322+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:57:26.876+0000] {processor.py:925} INFO - DAG(s) 'airflow_dag_amazon' retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:57:26.911+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:57:26.911+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-14T18:57:26.931+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:57:26.931+0000] {dag.py:4180} INFO - Setting next_dagrun for airflow_dag_amazon to None, run_after=None
[2025-01-14T18:57:26.960+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.651 seconds
[2025-01-14T18:57:57.314+0000] {processor.py:186} INFO - Started process (PID=554) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:57:57.316+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:57:57.329+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:57:57.328+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:57:58.084+0000] {processor.py:925} INFO - DAG(s) 'airflow_dag_amazon' retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:57:58.118+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:57:58.117+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-14T18:57:58.137+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:57:58.136+0000] {dag.py:4180} INFO - Setting next_dagrun for airflow_dag_amazon to None, run_after=None
[2025-01-14T18:57:58.166+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.878 seconds
[2025-01-14T18:58:28.428+0000] {processor.py:186} INFO - Started process (PID=562) to work on /opt/airflow/dags/dag_v1.py
[2025-01-14T18:58:28.430+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_v1.py for tasks to queue
[2025-01-14T18:58:28.433+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:58:28.432+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:58:29.037+0000] {processor.py:925} INFO - DAG(s) 'airflow_dag_amazon' retrieved from /opt/airflow/dags/dag_v1.py
[2025-01-14T18:58:29.067+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:58:29.067+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-01-14T18:58:29.086+0000] {logging_mixin.py:190} INFO - [2025-01-14T18:58:29.085+0000] {dag.py:4180} INFO - Setting next_dagrun for airflow_dag_amazon to None, run_after=None
[2025-01-14T18:58:29.110+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_v1.py took 0.691 seconds
